test/test.ml

```diff
@@ -4,32 +4,32 @@ open Printf
 let compile file pkgs =
   Cmd.(v "ocamlbuild" % "-use-ocamlfind" % "-tag" % "thread" % "-pkgs" % pkgs % (file ^ ".native"))
   |> OS.Cmd.run_out |> OS.Cmd.to_string ~trim:true |>
-    function | Ok r -> prerr_endline r; r | Error (`Msg e) -> raise (Failure (sprintf "Error: %s" e))
+    function | Result.Ok r -> prerr_endline r; r | Result.Error (`Msg e) -> raise (Failure (sprintf "Error: %s" e))
```

Comment:
could you at the top of the file do an `open Result` instead of having `Result` explicitly all over, please?

The reason is, once we decide to drop 4.02 support, we can remove the `result` package dependency, and no longer need to `open Result` (since the type result is part of Pervasives in 4.03+).